core/o11y/src/lib.rs

```diff
@@ -459,10 +460,13 @@ pub fn reload_log_config(config: Option<&log_config::LogConfig>) {
     };
     match result {
         Ok(_) => {
-            println!("Updated the logging layer according to `log_config.json`");
+            info!("Updated the logging layer according to `log_config.json`");
         }
         Err(err) => {
-            println!("Failed to update the logging layer according to the changed `log_config.json`. Errors: {:?}", err);
+            error!(
```

Comment:
Let's use `warn` instead of `error`, as it is not a critical issue to raise an error on.